custom/icds_reports/migrations/sql_templates/update_tables29.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE ccs_record_monthly ADD COLUMN valid_visits smallint;
+ALTER TABLE agg_ccs_record ADD COLUMN valid_visits smallint;
+ALTER TABLE agg_ccs_record ADD COLUMN expected_visits smallint;
```

Comment:
I think that you will want this to be an integer. It's likely that a state would have > 32k valid visits in a month. also you added it as integer in the models file